src/Config/ObjectModel/NestedMutationOptions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+/// <summary>
+/// Class that holds the options for all nested mutation operations.
+/// </summary>
+/// <param name="NestedCreateOptions">Options for nested create operation.</param>
+public class NestedMutationOptions
+{
+    // Options for nested create operation.
+    [JsonPropertyName("insert")]
```

Comment:
why is the json property name insert?

Reply:
Good catch.

Actually, the `JsonPropertyName` is not needed since we have custom converters. Removed this.